app/views/vaccines/index.html

```diff
@@ -22,6 +22,33 @@ <h1 class="nhsuk-heading-xl">Vaccines</h1>
     </div>
   </div>
 
+  {% from 'select/macro.njk' import select %}
+
+  {{ select({
+    id: "select-1",
+    name: "select-1",
+    label: {
+      text: "Select site"
+    },
+    items: [
+      {
+        value: 1,
+        text: "NHS.UK frontend option 1"
+      },
+      {
+        value: 2,
+        text: "NHS.UK frontend option 2",
+        selected: true
+      },
+      {
+        value: 3,
+        text: "NHS.UK frontend option 3",
+        disabled: true
+      }
+    ]
+  }) }}
+
+
```

Comment:
did you mean to add this?

Reply:
After speaking with Ileri to add/back in yes, but may need to add the vaccines as default